src/ui/MainWindow.cc

```diff
@@ -183,8 +183,10 @@ MainWindow::MainWindow(QSplashScreen* splashScreen)
     // Qt 4 on Ubuntu does place the native menubar correctly so on Linux we revert back to in-window menu bar.
     // TODO: Check that this is still necessary on Qt5 on Ubuntu
 #ifdef Q_OS_LINUX
+#ifndef __android__
     menuBar()->setNativeMenuBar(false);
 #endif
+#endif
     

```

Comment:
This is another experiment. By setting this, or not setting as it is the case, the main menu is shown natively. I personally don't like it as it wastes a lot of space. I want to integrate the main menu with the main tool bar. The idea is to have a little button that you *slide* and the menu slides into view from the side (the standard with mobile devices). I tried that and the logic works. What doesn't work is using QML dialogs as this don't seem to like not having a real QML Application is the main root.